### PR TITLE
Migrate toBounds method to be a static fromLngLat method in LngLatBounds

### PR DESCRIPTION
**Pull request by zhangyiatmicrosoft**
## Launch Checklist

- Add types to LngLatBounds constructor.
- remove toBounds
- Create a static method called fromLngLat(lnglat, radius).
- Fail build on all warnings to stop all future circular dependency problems

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license)
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.
 - [x] Write tests for all new functionality.
 - [x] Document any changes to public APIs.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: src/geo/lng_lat_bounds.ts [hunk 1]
```diff
@@ -23,16 +22,32 @@ class LngLatBounds {
...
+     * @example
+     * var sw = new maplibregl.LngLat(-73.9876, 40.7661);
+     * var ne = new maplibregl.LngLat(-73.9397, 40.8002);
+     * var llb = new maplibregl.LngLatBounds(sw, ne);
+     * OR
+     * var llb = new maplibregl.LngLatBounds([-73.9876, 40.7661, -73.9397, 40.8002]);
+     * OR
+     * var llb = new maplibregl.LngLatBounds([sw, ne]);
+     */
+    constructor(sw?: LngLatLike | [number, number, number, number] | [LngLatLike, LngLatLike], ne?: LngLatLike) {
```
Comment (HarelM) [inline src/geo/lng_lat_bounds.ts:39]: LangLatBoundsLike?

Comment (zhangyiatmicrosoft) [reply src/geo/lng_lat_bounds.ts:39]: LangLatBoundsLike includes LangLatBounds, which is not currently accepted by the ctor. Probably should not be added either

Comment (HarelM) [reply src/geo/lng_lat_bounds.ts:39]: Right.

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
Why didn't the test run? Can you do a dummy commit?

**Comment by zhangyiatmicrosoft** [comment]
> Why didn't the test run? Can you do a dummy commit?

Merged and now should work

**Pull request merged**